Makefile

```diff
@@ -399,32 +399,29 @@ ci-clean:
 
 .PHONY: ci-rp
 ci-rp: fix-macos-vendor
-	podman $(PODMAN_REMOTE_ARGS) \
-		build . \
+	docker build . $(DOCKER_BUILD_CI_ARGS) \
 		-f Dockerfile.ci-rp \
 		--ulimit=nofile=4096:4096 \
 		--build-arg REGISTRY=$(REGISTRY) \
 		--build-arg ARO_VERSION=$(VERSION) \
 		--no-cache=$(NO_CACHE) \
+		--target=builder \
+		-t $(LOCAL_ARO_RP_BUILD_IMAGE):$(VERSION)
+
+	docker build . $(DOCKER_BUILD_CI_ARGS) \
+		-f Dockerfile.ci-rp \
+		--ulimit=nofile=4096:4096 \
+		--build-arg REGISTRY=$(REGISTRY) \
+		--build-arg ARO_VERSION=$(VERSION) \
+		--cache-from=$(REGISTRY)/$(LOCAL_ARO_RP_BUILD_IMAGE) \
```

Comment:
I think that we will want `$(LOCAL_ARO_RP_BUILD_IMAGE)` instead of `$(REGISTRY)/$(LOCAL_ARO_RP_BUILD_IMAGE)` - we want to use the local images as a cache - the one we just built, rather than something remote.

Reply:
Yeah right ,Thanks for the feedback! I have updated the code to use `$(LOCAL_ARO_RP_BUILD_IMAGE) `instead of `$(REGISTRY)/$(LOCAL_ARO_RP_BUILD_IMAGE)`